crates/sui-config/src/node.rs

```diff
@@ -59,6 +62,10 @@ fn default_metrics_address() -> SocketAddr {
     SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 9184)
 }
 
+pub fn default_admin_interface_port() -> u16 {
+    1337
```

Comment:
why is this a function call instead of a config value?

Reply:
It is a config value, this is just the default in the event it wasn't specified in the config